README.md

```diff
@@ -43,32 +43,34 @@ An app for all those who care about their health.
 │   └───images
 |   └───favicon
 ├───script
-│   ├───jquery.min.js              
-│   └─── quiz.js                
+│   ├───jquery.min.js
+│   └─── quiz.js
 |   └─── script.js
 
 ```
 
-  
-
 <!--
 ##  ▶️ How to Run the Project
 **1.** Navigate to the project directory.
 -->
 
-
-
-<h1 align=center> Project Admin </h1>
-
+<h1 align=center> 👨‍💻 Project Admin </h1>
 
   <div align="center">
 <table>
 <tr>
 
 <td align="center"><a href="https://github.com/ghulamyazdani"><img src="https://avatars.githubusercontent.com/u/55938346?v=4" width=150px height=150px /></a></br> <h4 style="color:white;">Ghulamyazdani</h4>
 
-     
 </tr>
 </table>
 <br>
 
+<br>
+
+<h1 align=left> 🙌 Contributors</h1>
```

Comment:
It would be better if you  center align it!!

Reply:
Done